CONTRIBUTING.md

```diff
@@ -15,8 +15,8 @@ $ circleci build --job checks
 ```
 
 The above job contains following checkers:
-- [flake8](http://flake8.pycqa.org)
 - [autopep8](https://github.com/hhatto/autopep8)
+- [flake8](http://flake8.pycqa.org)
```

Comment:
I think the contents of this file are identical to those in master branch.
Please revert this file to master branch version.